pkg/test/framework/components/istio/operator.go

```diff
@@ -217,14 +214,38 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 			return nil, err
 		}
 	}
+	// If the control plane cluster is not the config cluster, we are under central Istiod which uses remote KUBECONFIG
+	// we follow below install sequences:
+	// install istiod Remote + base in remote cluster first.
+	// install istiod in control cluster
```

Comment:
There is no defined term "control cluster" ... it should be  "primary cluster"

Reply:
fixed